assets/js/tomSelect.js

```diff
@@ -0,0 +1,5 @@
+const TomSelect = require('tom-select');
+
+[...document.getElementsByClassName('tom-select')].forEach(select =>
```

Comment:
why don't you use:

```suggestion
document.getElementsByClassName('tom-select').forEach(select =>
```

Reply:
It's not possible to forEach on a HTMLCollection, which is what `getElementsByClassName` returns. We could use a for/of instead however, but for some reason it throws errors in the console.
We also could use a QuerySelector if you don't like what I did here.

Reply:
ok ok :)